interface/forms/eye_mag/save.php

```diff
@@ -386,9 +406,14 @@
                 $content = substr($content, 0, $i + 6) . $webserver_root . substr($content, $i + 6 + $wrlen);
             }
         }
-
-        $pdf->writeHTML($content, false);
-        $temp_filename = '/tmp/'.$filename;
+        // Below is for including style sheet for report specific styles. Left here for future use.
+        //$styles = file_get_contents('../css/report.css');
+        //$pdf->writeHTML($styles, 1);
+        //$pdf->writeHTML($content, 2);
+
+        $pdf->writeHTML($content, false); // false or zero works for both mPDF and HTML2PDF
+        $tmpdir = $GLOBALS['OE_SITE_DIR'] . '/documents/temp/'; // Best to get a known system temp directory to ensure a writable directory.
```

Comment:
Is this file getting removed after it's used?

Reply:
Nope, thats why I sent to system. I'm not even sure it needs to get saved at all.

Reply:
Very odd. It should not linger (note the filename is not very specific, so could chance it would get overwritten).  It looks like it's supposed to store it has a document, but it doesn't seem to work on the testing I just did.

Reply:
@sjpadgett ,
It appears you may be getting sucked into another OpenEMR vortex. I wish you the best :)
-brady

Reply:
I'll delete them. just tested..need a minute

Reply:
Oh well, this is a very nice module so worth the effort.